generated/nidaqmx/_task_modules/ci_channel_collection.py

```diff
@@ -834,14 +778,14 @@ def add_ci_pulse_width_chan(
             custom_scale_name=""):
         """
         Creates a channel to measure the width of a digital pulse.
-        **starting_edge** determines whether to measure a high pulse or
-        low pulse. With the exception of devices that support multi-
-        counter tasks, you can create only one counter input channel at
-        a time with this function because a task can contain only one
-        counter input channel. To read from multiple counters
-        simultaneously, use a separate task for each counter. Connect
-        the input signal to the default input terminal of the counter
-        unless you select a different input terminal.
+        **edge** determines whether to measure a high pulse or low
```

Comment:
I don't even know why this is bolded, since its not a parameter... so yeah, this is fine.

Reply:
This has been fixed now, there is no change in the file